types/storybook__addon-options/storybook__addon-options-tests.ts

```diff
@@ -4,6 +4,20 @@ setOptions({
   name: 'My Storybook',
   url: 'https://example.com',
   goFullScreen: false,
+  showStoriesPanel: false,
+  showAddonPanel: false,
+  showSearchBox: false,
+  addonPanelInRight: false,
+  sortStoriesByKind: false,
+  hierarchySeparator: /\//,
+  hierarchyRootSeparator: /\|/,
+  selectedAddonPanel: 'storybook/actions/action-panel',
+});
+
+setOptions({
+  name: 'My Storybook - deprecated options',
+  url: 'https://example.com',
```

Comment:
url and goFullScreen are not deprecated options - so consider removing them from this section